assets/src/infrastructure/ui/layout/modal/modalCloseButtonProps/styles.scss

```diff
@@ -5,7 +5,6 @@
 		position: absolute;
 		top: 8px;
 		right: 12px;
-		min-width: auto;
```

Comment:
related to this width:

![image](https://user-images.githubusercontent.com/1477453/82354558-a31a1480-9a09-11ea-891b-6958590f3647.png)